shard_static/models.py

```diff
@@ -1,25 +1,29 @@
 from django.db import models
 
+from shard_static.constants import ALL_SHARD_GROUP
 from shard_static.managers import ShardStaticManager
 from shard_static.mixins import ShardStaticMixin
 
 
-class StaticSyncStatus(ShardStaticMixin, models.Model):
+class BaseStaticSyncStatus(models.Model):
     static_model_key = models.CharField(max_length=64, unique=True, verbose_name='Static Model Key')
-    last_modified = models.DateTimeField(null=True, verbose_name='Last Modified', )
-
-    diffusible = False
 
+    shard_group = ALL_SHARD_GROUP
     objects = ShardStaticManager()
 
     class Meta:
-        db_table = 'static_sync_status'
+        abstract = True
 
+    def set_revision(self, shard_static_model):
```

Comment:
BaseStaticSyncStatus -> shard_static_model -> BaseShardStaticModel -> status .....
Is there better way?

Reply:
how about `record_status` for function name.

Reply:
need command about check new value